docs_user/modules/proc_deploying-a-ceph-ingress-daemon.adoc

```diff
@@ -2,16 +2,24 @@
 
 = Deploying a {Ceph} ingress daemon
 
-To match the cardinality diagram, you use cephadm labels to refer to a group of nodes where a given daemon type should be deployed. For more information about the cardinality diagram, see xref:ceph-daemon-cardinality_migrating-ceph[{Ceph} daemon cardinality].
-`HAProxy` is managed by {OpenStackPreviousInstaller} through `Pacemaker`: the three running instances at this point will point to the old RGW backends, resulting in a broken configuration.
-ifeval::["{build}" != "upstream"]
-Since you are going to deploy the Ceph ingress daemon, the first thing to do is remove the existing `ceph_rgw` config, clean up the config created by {OpenStackPreviousInstaller} and restart the service to make sure other services are not affected by this change.
-endif::[]
-ifeval::["{build}" != "downstream"]
-Since you are going to deploy the https://github.com/openstack-archive/tripleo-ansible/blob/stable/wallaby/tripleo_ansible/ansible_plugins/modules/ceph_mkspec.py[Ceph ingress daemon], the first thing to do is remove the existing `ceph_rgw` config, clean up the config created by {OpenStackPreviousInstaller} and restart the service to make sure other services are not affected by this change.
-endif::[]
-After you complete this procedure, you can reach the RGW backend from the ingress daemon and use RGW through the {object_storage} command line interface (CLI).
-//kgilliga: I will rewrite this intro for GA.
+To deploy the Ceph ingress daemon, you perform the following actions:
+
+. Remove the existing `ceph_rgw` configuration.
+. Clean up the configuration created by {OpenStackPreviousInstaller}.
+. Restart the service to ensure that other services that consume the {object_storage_first_ref} are not affected by this change.
```

Comment:
I suggest to change L9 from `Restart the service ....` to: `Redeploy the {object_storage_first_ref} service`.